src/App.vue

```diff
@@ -206,8 +232,9 @@ class App extends Vue {
 
         try {
             const cashlink: Cashlink = await hubApi.createCashlink({
-                appName: 'Nimiq Gift Card Creator',
+                appName: 'Nimiq Gift Card',
```

Comment:
```suggestion
                appName: 'Nimiq Cards',
```
Because the term "gift cards" is already used (see nimiqgiftcards.com ;) ) - the idea was to stick with "cards" or "Nimiq Cards" depending on the context.

Reply:
This is used in the "Back to <appname>" link in the bottom of the Hub, so I think it's ok to call it whatever is helpful in that context. The app has a header of "Nimiq Gift Cards", so it's already called that.